app/models/reminder.js

```diff
@@ -0,0 +1,5 @@
+import DS from 'ember-data'
+
+export default DS.Model.extend({
+  text: DS.attr('string')
```

Comment:
There should be more properties in the model than 'text' - what else does a reminder need to have?